src/mixins/planningActionMixin.js

```diff
@@ -275,6 +275,8 @@ export const planningActionMixin = {
           })
             .onOk(this.onOkForEventCreation)
             .onCancel(this.onCancelForEventCreation);
+        } else {
```

Comment:
il faut que tu mettre un else if sur le `if (this.newEvent.auxiliary && this.$_.get(this.newEvent, 'repetition.frequency', '') !== NEVER)` car sion quand tu crées une absence, ca la crées deux fois